### ISSUE TITLE
Rework health service push changes from 8019 to block mental health care and 

### ISSUE THREAD

**Issue by JayDarnell** [department-of-veterans-affairs/va.gov-cms#10454]
## Description

Reworks PR #8036 

Instead of pushing ALL services we want to push all services except for:

- 43 - Mental health care
- ?

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by dsinla** [comment]
@JayDarnell would it be correct to say this is blocked, waiting on v1?

**Comment by jilladams** [comment]
Noted on PR: this ticket needs an update for current status. It's been riding sprint to sprint, but I'd propose pulling out of sprint until it's clear what the steps are to get it merged / complete and that's a priority. (cc @dawnpruitt )

**Comment by JayDarnell** [comment]
@jilladams @dsinla  this ticket is done. We are basically in a holding pattern until the lighthouse team fixes their latency issue and gives us the greenlight to merge.

**Comment by jilladams** [comment]
Noting convo David and I had yesterday: one option in future when work is completed in a sprint but not yet mergeable:  merge your PR into an integration branch, so you can get code review / close the current-sprint ticket. A sep ticket can track  merging the integration branch to main in a future sprint. (We on PW usually make creating that follow up ticket an AC, so it doesn't get lost.)

**Comment by JQuaggles** [comment]
@dawnpruitt / @JayDarnell 
Can we take a look at doing the above that Jill mentioned for this? (Or has it already been completed?)

**Comment by JayDarnell** [comment]
These Lighthouse tickets are a bit unique. Long story short, we had multiple lighthouse updates in the works, some that were dependent on V1 (new version of their API) when they froze code updates due to a latency issue for 6-7 weeks. Because each update needs to be independently verified with the Lighthouse team in a staging environment before being merged into production I've avoided merging them into a single integration branch. 

All of that being said, this has not been completed yet. The code freeze was lifted last week and now we're getting ready to dust these off again (In fact, these could be looked at this sprint)

**Comment by jilladams** [comment]
Work is complete, in an integration branch, will close on V1 ship.

**Comment by swirtSJW** [comment]
There is no explanation in this ticket to explain why.   Trying to sort that out [here in slack](https://dsva.slack.com/archives/C0FQSS30V/p1691617217113299?thread_ts=1691617091.325469&cid=C0FQSS30V).

**Comment by jilladams** [comment]
Re: 43 - Mental health care / 48 - Caregiver support & why excluded: CMS is not the source of truth for Mental Health yet, and if we pushed, we would be acting as SOT.

@mmiddaugh to confirm with Dave Conlon.

**Comment by jilladams** [comment]
Closing because the work is complete in Int branch, but we do need the closure on question above.

**Comment by mmiddaugh** [comment]
Per Adam with Lighthouse: 

> Caregiver is populated from a local text file baked into the application. It was given to us by CARMA two years ago and is just a list of facility ids they said have the service.

**Comment by swirtSJW** [comment]
This code went to prod today.

**Comment by jilladams** [comment]
We push Mental Health phone numbers from Drupal, as of recent cutover. We did not, at that time, discuss pushing Mental Health service information, and still do not push that today.
